src/System.IO.FileSystem/tests/Directory/Directory_EnumerableAPIs.cs

```diff
@@ -3,742 +3,234 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.IO;
-using System.Security;
+using System.Linq;
 using Xunit;
 
 namespace EnumerableTests
 {
-    public class Directory_EnumerableTests
+    public class Directory_EnumerableTests : IClassFixture<TestFileSystemEntries>
```

Comment:
Closing PR for now but will prepare new one)
BTW `IClassFixture` support that is really needed.